tools/skywalker_diff.py

```diff
@@ -13,9 +13,13 @@
 import numpy.linalg as lin
 
 def norms(vals) :
-    L1 = lin.norm(vals,1)
-    L2 = lin.norm(vals)            
-    Linf = lin.norm(vals,np.inf)            
+    if 1 == vals.ndim :
```

Comment:
Do we know that `vals` is a numpy array here?

Reply:
Yes, at line 66 vals is defined as 
```
        vals1 = np.array(vars1[key])
        vals2 = np.array(vars2[key])
```